test_tar_changes.py

```diff
@@ -41,6 +41,7 @@ def distgitdir(tmpdir):
 
 
 def test_commit_distgit_amend(distgitdir):
+    # Test commit_distgit_amend()
```

Comment:
A comment like `Test commit_distgit_amend()` is not adding any value - the method name is literally `test_commit_distgit_amend()`.

If there's something a casual reader would need to know beyond the method name, then let's add it here. Otherwise, we don't need to add anything.